### ISSUE TITLE
save-then-deploy, increase lambda default timeout (two small fixes)

### ISSUE THREAD

**Issue by jeffknerr** [itz-mathesh/serverless-ec2-scheduler#1]
Hi Mathesh. Good tutorial. I just worked through it and only got stuck on two small issues:
- had to File>Save the lambda function before I hit the Deploy button
- had to Edit the lambda default config to change the timeout from 3sec to 15sec
Otherwise everything worked perfectly. Thanks!

**Comment by mathesh-me** [comment]
Hello Jeffknerr, I wanted to express my gratitude for identifying the issues.
* Just a friendly reminder, there's no need to save the file before deploying. In Lambda, the Deploy option is specifically for saving the code.
* Also, I realized I forgot to mention the runtime of the function. For the runtime, we typically adjust the option to be between 10 to 15 seconds. Now I updated the README.md file.
Thank you once again for your attention. Let me know if there's anything else I can clarify or help with.
